packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -1073,4 +1145,146 @@ The Authenticator object defines the following properties:
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Authenticator Method object
+
+#### Authenticator Method properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `status` | String (Enum) | The status of the authenticator method. Possible values: `ACTIVE` or `INACTIVE` | All |
+| `type` | String (Enum) | The type of authenticator method. Possible values: `cert`, `duo`, `email`, `idp`, `otp`, `password`, `push`, `security_question`, `signed_nonce`, `sms`, `totp`, `voice`, or `webauthn`  |  All |
+| `_links` | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this Authenticator Method object  |  All |
+| `settings` | [Authenticator Method Settings object](#authenticator-method-settings-object) | Specific settings for the authenticator method  | `otp`, `push`, `signed_nonce`, `totp`, `webauthn` |
+| `verifiableProperties` | Array of strings (Enums) | Verifiable properties. Possible values: `DEVICE_BOUND`, `HARDWARE_PROTECTED`, `PHISHING_RESISTANT`, `USER_PRESENCE`, or `USER_VERIFYING`  | `cert`, `duo`, `idp` |
+
+### Authenticator Method Settings object
+
+#### Authenticator Method Settings properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `algorithms` | String (Enum) | Algorithms supported. Possible values: `RS256`, `ES256`, `HMACSHA1`, `HMACSHA256`, or `HMACSHA512` | `otp`, `totp`, `push`, `signed_nonce` |
+| `attachment` | String (Enum) | Method attachment. Possible values: `ANY`, `BUILT_IN`, or `ROAMING`  | `webauthn` |
+| `keyProtection` | String (Enum) | Key protection. Supported values: `ANY` or `HARDWARE`  | `totp`, `push`, `signed_nonce`
+| `protocol` | String (Enum) | The protocol used. Possible values: `SYMANTEC`,`TOTP`, or `YUBICO` | `otp` |
+| `timeIntervalInSeconds` | Integer | Time interval for TOTP in seconds | `totp` |
+| `encoding` | String (Enum) | Encoding of OTP value. Possible values: `Base32`, `Base64`, or `Hexadecimal`  | `totp` |
+| `passCodeLength` | Integer | Number of digits in an OTP value  | `otp`, `totp` |
+| `acceptableAdjacentIntervals` | Integer | Number of acceptable adjacent intervals. Possible values: 0&ndash;10 | `otp` |
+| `showSignInWithOV` | String (Enum) | Shows sign-in with Okta Verify. Possible Values: `ALWAYS` or `NEVER`  | `signed_nonce` |
```

Comment:
I'm not sure what you mean by "Shows sign-in with Okta Verify." So, I don't know how to edit the fact that sign-in is hyphenated and yet has no noun associated with it.....Can I get a little more context on this one? :)

Reply:
I updated descriptions for acceptableAdjacentIntervals, showSignInWithOV, keyProtection, encoding. Hopefully, this will add more context. @susanharper-okta please review again 🙏